src/coreclr/inc/clrconfigvalues.h

```diff
@@ -310,6 +310,7 @@ CONFIG_DWORD_INFO(INTERNAL_JitGCStress, W("JitGCStress"), 0, "GC stress mode for
 CONFIG_DWORD_INFO(INTERNAL_JitHeartbeat, W("JitHeartbeat"), 0, "")
 CONFIG_DWORD_INFO(INTERNAL_JitHelperLogging, W("JitHelperLogging"), 0, "")
 RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_JITMinOpts, W("JITMinOpts"), 0, "Forces MinOpts")
+RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_JITAllowOptionalRelocs, W("JITAllowOptionalRelocs"), 1, "Allow optional relocs")
```

Comment:
We really shouldn't use `JIT` (all capital letters) in environment variables. `JITMinOpts` (and related) is the unfortunate exception; almost every other use is `Jit` (mixed case).

The name and text "Allow Optional Relocs" isn't too descriptive. I don't know what it means when I see it.

Reply:
I can change casing - I indeed just copied JITMinOpts, regarding the name - I didn't want to use "DisableRelocs" like name because we can't physically disable them for all cases (e.g. AOT), if you have a better name - I can change that